docs/architecture-decision-records/20220121-conduit-plugin-architecture.md

```diff
@@ -86,7 +86,7 @@ Functions will be called in the order in which they are defined.
   and* all remaining acknowledgments are received (this is handled by the SDK).
 - `Stop` - Conduit signals to the plugin it should stop fetching new records from the origin. The plugin is still
   allowed to produce records in the response stream of `Run` in case there are any cached records in memory, but it
-  should not go and fetch new ones from the origin.
+  should not Go and fetch new ones from the origin.
```

Comment:
I think this one would still be `go` since the verb is used here, instead of the noun referring to the programming language:

```suggestion
  should not go and fetch new ones from the origin.
```

Reply:
Lol, didn't notice that 😅 Good points, I'll remove that rule and revert these words.